### DISCUSSION TITLE
Popup filteroption: Enable only player releated popups

### DISCUSSION THREAD

**Discussion by u1ps** [etlegacy/etlegacy#1911] [Ideas]
Right now its possible through options to filter "Death" popups. On a big server there are so much popups in sec that it gets very distracting. So the filter is on but what i find myself doing often is opening console to see what killed me. What about another filter option where player deaths will remain in popup are but all other is filtered

**Comment by Aciz** [comment]
I've actually had an idea of implementing some sort of popup highlighting option, similar to CS:GO for example, where kill popups that involve you would be highlighted with a rectangle or something.

![image](https://user-images.githubusercontent.com/14221121/153750557-293e80d1-b952-4320-992c-b2005d5e6c6b.png)
